### DISCUSSION TITLE
TypeScript friendly adaptation

### DISCUSSION THREAD

**Discussion by AH-dark** [hexojs/hexo#4983] [Question / Answer] [answered]
Is there anyone who can work on perfecting @types/hexo or adding a more complete TypeScript declare to hexo? Manipulating Hexo and Hexo Database in TypeScript is a torture due to version lag!

**Comment by yoshinorin** [comment, marked as answer]
@AH-dark 
Thank you for your comment :)
Now we are working replace to TS. Also, PR are welcome.

> https://github.com/hexojs/warehouse/pull/60
> https://github.com/hexojs/hexo-fs/pull/66

**Reply by AH-dark** [reply]
Sorry, I'd like to do that but I do not have enough ability, looking forward to your results!
